local/docker-compose.yml

```diff
@@ -6,14 +6,14 @@ services:
       - "8125:8125/udp"
       - "8126:8126/tcp"
     environment:
+      - DD_ENV=local
+      - DD_SITE=datadoghq.com
+      - DD_APM_ENABLED=true
+      - DD_APM_NON_LOCAL_TRAFFIC=true
       - DD_API_KEY
-      - DD_ENV
-      - DD_SITE
-      - DD_APM_ENABLED
-      - DD_APM_NON_LOCAL_TRAFFIC
-      - DD_LOGS_ENABLED
-      - DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL
-      - DD_CONTAINER_EXCLUDE
+      - DD_LOGS_ENABLED=true
+      - DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL=true
+      - DD_CONTAINER_EXCLUDE=image:gcr.io/datadoghq/agent
```

Comment:
Think we can also add `DD_TRACE_ENABLED=0` (or 1) here just since a lot of the other env vars are here?

Reply:
I think it makes more sense where it's already at, which is the service (as opposed to the agent).

Reply:
Ah whoops, my bad missed that